data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditableAuthorisedDataRequest.java

```diff
@@ -22,16 +22,16 @@
 import java.util.StringJoiner;
 
 /**
- * The class contains the authorised access to the resource.  This class is a container for the original data,
- * {@link DataRequest}, the reference to the data that is authorised to be seen, {@link AuthorisedDataRequest} and
- * {@link AuditErrorMessage} processing.  The container will hold the {@code AuditErrorMessage} if there was an error
- * in getting the authorised data.
+ * The class contains the data available after the request to authorise the access to the data. This will include the
+ * data request, {@link DataRequest}, the reference to the data that is authorised to be seen,
+ * {@link AuthorisedDataRequest}, and {@link AuditErrorMessage} processing if there was an error in requesting the
+ * authorisation.
```

Comment:
```suggestion
 * {@link AuthorisedDataRequest}, and if there was an error when requesting the authorisation, {@link AuditErrorMessage}
```

Reply:
Has this been missed?

Reply:
This was done.  Something strange with the GitHub view of the code.